src/TabViewNativeComponent.ts

```diff
@@ -23,6 +23,16 @@ export interface TabViewProps extends ViewProps {
   labeled?: boolean;
   sidebarAdaptable?: boolean;
   scrollEdgeAppearance?: string;
+  activeTintColor?: ProcessedColorValue | null;
+  inactiveTintColor?: ProcessedColorValue | null;
 }
 
+export type TabViewNativeProps = Omit<
+  TabViewProps,
+  'activeTintColor' | 'inactiveTintColor'
+> & {
+  activeTintColor?: ColorValue;
+  inactiveTintColor?: ColorValue;
+};
+
```

Comment:
I'm afraid this may fail during codegen with new architecture. 

Can you try installing pods with RCT_NEW_ARCH_ENABLED=1 to check if codegen generates specs correctly?